fiducial_slam/include/fiducial_slam/map.h

```diff
@@ -92,7 +92,7 @@ class TransformWithVariance {
     }
 
     // Used to combine this transform with another one keeping variance the same
-    TransformWithVariance& operator*=(const tf2::Transform& rhs) {
+    void operator*=(const tf2::Transform& rhs) {
```

Comment:
This is valid and legal, but it is convention to return a `T&` and then return `*this`.

Reply:
The way it was causes a compiler warning.   If you know how to make follow conventions and not cause a warning, feel free.